hadoop-ozone/tools/src/main/java/org/apache/hadoop/ozone/freon/OzoneClientKeyGenerator.java

```diff
@@ -93,6 +101,8 @@ public Void call() throws Exception {
 
     contentGenerator = new ContentGenerator(keySize, bufferSize);
     metadata = new HashMap<>();
+    replicationConfig = ReplicationConfig.parse(replicationType, replication,
```

Comment:
parse is returning default values even if we pass replication and type as null.
When we set the EC on bucket, ideally it's not mandatory to pass the type and replication with key as we moved the default configs to server side along with bucket.

Please refer the JIRA: HDDS-6184